src/modules/Admin/components/PortalUserList/Table/ActionsToolbar/container.js

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2017 Observational Health Data Sciences and Informatics
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Company: Odysseus Data Services, Inc.
+ * Product Owner/Architecture: Gregory Klebanov
+ * Authors: Anton Gackovka
+ * Created: May 23, 2018
+ */
+
+import actions from 'actions/index';
+import { ContainerBuilder, Utils } from 'services/Utils';
+import ActionsToolbar from './presenter';
+import selectors from 'modules/Admin/components/PortalUserList/Table/selectors';
+import { batchOperationType } from 'modules/Admin/const';
+
+class UserListActionsToolbarBuilder extends ContainerBuilder {
+
+  getComponent() {
+    return ActionsToolbar;
+  }
+
+  mapStateToProps(state) {
+    const { pathname, query } = state.routing.locationBeforeTransitions;
+    return {
+      selectedUsers: selectors.getSelectedUsers(state),
+      query,
+      pathname,
+    };
+  }
+
+  getMapDispatchToProps() {
+    return {
+      batch: actions.adminSettings.portalUserList.batch,
+      loadUsersWithCurrentQuery: (query) => actions.adminSettings.portalUserList.query({ query }),
```

Comment:
`loadUsersWithCurrentQuery` means load with _current_ query, not with any. Such function could be created in mergeProps. But this one is just loadUsers. Same story below in several places